ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PointSchedule/Shrinking.hs

```diff
@@ -60,6 +73,68 @@ shrinkOtherPeers shrink Peers{honest, others} =
   map (Peers honest . Map.fromList) $
     shrinkList (traverse (traverse shrink)) $ Map.toList others
 
+-- | Shrinks an honest peer by removing ticks.
+-- Because we are manipulating `PeerSchedule` at that point, there is no proper
+-- notion of a tick. Instead, we remove points of the honest `PeerSchedule`,
```

Comment:
```suggestion
-- | Shrinks an honest peer by removing ticks.
-- Because we are manipulating 'PeerSchedule' at that point, there is no proper
-- notion of a tick. Instead, we remove points of the honest 'PeerSchedule',
```